www/apps/docs/content/deployments/server/deploying-on-heroku.mdx

```diff
@@ -146,15 +146,15 @@ However, if you use another add-on, make sure to set the environment variable `D
 
 #### (Optional) Configure Modules Environment Variables
 
-If you use in your Medusa backend modules that require setting environment variables, then you should set them at this point.
+If you use any code in your Medusa backend modules that require setting environment variables, then you should set them at this point.
```

Comment:
It's not a typo in the original text, but I guess the wording isn't clear. Maybe change it to:

```suggestion
If you use modules in your Medusa backend that require setting environment variables, then you should set them at this point.
```